utils/cosmos/iscn/sign.ts

```diff
@@ -25,26 +25,48 @@ export function formatISCNTxPayload(payload: ISCNRegisterPayload): ISCNSignPaylo
     authorNames,
     authorUrls,
     authorWallets,
+    likerIds,
+    descriptions,
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     cosmosWallet,
     ...data
   } = payload;
+
   const contentFingerprints = []
   if (fileSHA256) contentFingerprints.push(`hash://sha256/${fileSHA256}`)
   if (ipfsHash) contentFingerprints.push(`ipfs://${ipfsHash}`)
-  const stakeholders = []
+  const stakeholders:any = []
   if (authorNames.length) {
     for (let i = 0; i < authorNames.length; i += 1) {
       const authorName = authorNames[i]
       const authorUrl = authorUrls[i]
-      const authorId = authorWallets[i]
+      let authorId = ''
+      const authorWalletAddress:any = []
+      const sameAs:any = []
+      authorWallets[i].forEach((a: any) => {
+        if (a.type === 'cosmos') {
+          authorId = `did:cosmos:${a.address.slice(6)}`
+          authorWalletAddress.push({ address: authorId, type: a.type })
+          sameAs.push(authorId)
+        } else if (a.type === 'eth') {
+          authorId = `did:eth:${a.address}`
+          authorWalletAddress.push({ address: authorId, type: a.type })
+          sameAs.push(authorId)
+        }
+      })
+      const likerId = `https://like.co/${likerIds[i]}`
```

Comment:
This should either be used as `url` or `sameAs` instead of a new key called `likerId`
The `url` would means the primary url that can identify the author, where `sameAs` are all the other urls/ids that connects to the same author